modules/apps/frontend-js/frontend-js-spa-web/test/app/App.ts

```diff
@@ -1251,36 +1405,37 @@ describe('App', function () {
 		userEvent.click(link, {button: 1});
 		userEvent.click(link, {button: 2});
```

Comment:
These lines 1401-1406 contain a 2nd parameter, but the `userEvent.click()` expects 1 parameter instead of 2.

What do we do here, can I safely delete these because it's supposed to not work as the method doesn't support the 2nd parameter? Do we leave it as is and do `// @ts-ignore`?

Alternatively, I can leave this one until the end until it all works and then see if this test fails without those 2nd parameters.